setup.py

```diff
@@ -33,7 +33,7 @@
     # Versions should comply with PEP440.  For a discussion on single-sourcing
     # the version across setup.py and the project code, see
     # https://packaging.python.org/en/latest/single_source_version.html
-    version='1.2.0',
+    version='1.2.2',
```

Comment:
Should this be `1.2.2.rc1`? Or is it intentional to go straight to 1.2.2? It depends on how we are planning to install test this.

Reply:
@alvinkrebs Check this before merging. I approved in case it was intentional.